src/aind_data_access_api/document_db.py

```diff
@@ -429,7 +429,10 @@ def aggregate_docdb_records(self, pipeline: List[dict]) -> List[dict]:
 
     def insert_one_docdb_record(self, record: dict) -> Response:
         """Insert one new record"""
-        if record.get("_id") is None:
+        is_managed_collection = (
```

Comment:
I think it might be better if this logic is handled by the API Gateway. Maybe we can get rid of this check here and just let the api gateway return an error code.